lib/facter/jenkins.rb

```diff
@@ -10,7 +10,6 @@
 Facter.add(:jenkins_plugins) do
```

Comment:
For compatibility I toyed with the idea to create a structured `jenkins` fact where `plugins` is a key, but I had issues testing it properly and then had a lack of time. I also thought it would be good to reduce the number of entries.
```ruby
Facter.add(:jenkins, type: :aggregate) do
  confine kernel: 'Linux'

  chunk :plugins do
    plugins = Puppet::Jenkins::Plugins.available
    plugins.transform_values { |plugin| plugin.slice(:plugin_version) }
  end
end
```